### DISCUSSION TITLE
Outlier and anomaly detection

### DISCUSSION THREAD

**Discussion by mihow** [milvus-io/milvus#22481] [Ideas & Feature requests]
Is it possible to query for dissimilar results? I can't find any discussion about this, but it seems like it would be a common use case when you have a database of vectors.

**Comment by jiaoew1991** [comment]
Hi @mihow, do you mean to search for topK most different vectors? Could you describe the detail of the scenario, I do not know how to usage it 😄

**Reply by xiaofan-luan** [reply]
I have one similar request here. User want to find special cases among all the vectors. This is widely used under anomaly detection and data annotations
